daisy_workflows/image_import/import_from_image.wf.json

```diff
@@ -41,7 +45,8 @@
         "Name": "${disk_name}",
         "Type": "pd-ssd",
         "ExactName": true,
-        "SourceImage": "${source_image}"
+        "SourceImage": "${source_image}",
+        "is_windows": "${is_windows}"
```

Comment:
Shouldn't this be: "IsWindows": "${is_windows}"?

Reply:
Yeah, I wasn't sure either since both naming formats are used in different JSON definitions. Now that you mention it, I think it makes sense to use `IsWindows` here, though, to at least match the rest of this block

Reply:
Would it even work with is_windows? The field name is IsWindows. Unless JSON unmarshalling is smart enough to convert between underscore and camel case notation, I don't see how this would work.

Reply:
Yeah, this is the syntax for specifying how the field appears in your json document:

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/896a2ffdf8a18584815433aa58e278deebdfed79/cli_tools/common/utils/logging/service/log_entry.go#L35

Reply:
This was probably an oversight from Tony. The rest of Daisy uses lower case camel case notation, we should not mix different notations for new code.

Reply:
Also, he might have been forced to use underscore notation because these jsons are sent to Firelog and they might have different notation.

For internal Daisy fields, we should keep camel case. You actually don't need to specify json field name, it will default to Go field name. Check other structs in Daisy.

Reply:
Changed to `isWindows`